### DISCUSSION TITLE
Need to update laravel laravel 10

### DISCUSSION THREAD

**Discussion by ShaungBhone** [rahulhaque/laravel-filepond#36] [Show and tell]
Please need to update laravel version. I got this [error](https://laracasts.com/discuss/channels/laravel/upgrading-laravel-9-to-10).

**Comment by rahulhaque** [comment]
@ShaungBhone I will asap. Thanks for informing.

**Comment by rahulhaque** [comment]
@ShaungBhone I've updated the dev branch to support L10. Can you please try and confirm? I'll be really busy for couple of days. Meanwhile use the dev version till I release new one. Install with `composer require rahulhaque/laravel-filepond:dev-dev`

**Reply by ShaungBhone** [reply]
Thank you let me try this again.

**Reply by rahulhaque** [reply]
Released a new version to support Laravel 10. Enjoy. :sparkles:
